app/components/accordion-pod/component.js

```diff
@@ -31,18 +33,13 @@ export default Component.extend(ManageLabels, {
       name:           'node',
       sort:           ['displayName'],
       translationKey: 'generic.node',
-    },
-    {
-      name:           'displayIp',
-      sort:           ['displayIp'],
-      translationKey: 'generic.ipAddress',
       width:          180
     },
   ],
 
-  expandAllObserve: function() {
-    let expandAll = this.get('expandAll');
+  expandAllObserve: observer('expandAll', function() {
```

Comment:
Could we alias this or just pass down the variable to expandAll? Doesn't seem like we need an observer for this.